src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -147,17 +147,19 @@ export class ConfigurationGroupComponent implements AfterViewInit {
 
   buildGroupOfItems(configItems) {
     configItems?.forEach(config => {
-      if (!config.hasOwnProperty('value')) {
-        config.value = config.default;
-      }
-      let isGroupNameExist = this.groups.some(obj => Object.values(obj.group).includes(config.displayName ? config.displayName : config.key));
-      let group = { key: config.key, name: config.displayName ? config.displayName : config.key, description: config.description };
-      if (isGroupNameExist) {
-        // If same group exist, create new group with coonfig key and the description of the configuration
-        group = { key: config.key, name: config.key, description: config.description }
+      if(config.readonly != 'true'){
```

Comment:
What will happen to readonly items?

Reply:
As per current design readonly items are not rendered on GUI.